GameData/RealismOverhaul/RO_SuggestedMods/MagicSmokeIndustries/RO_IR.cfg

```diff
@@ -0,0 +1,1257 @@
+@PART[IR_Rotatron_Bearing_Uncontrolled_v3]:FOR[RealismOverhaul]
+{
+        %RSSROConfig = True
+        @mass = 0.006
+	%TechRequired = start
+	%cost = 5
+	%entryCost = 5
+	RP0conf = true
```

Comment:
RO configs are not allowed to configure parts for RP-1. That part needs to be done in RP-1 itself using the Part Browser webapp.

Reply:
That makes sense. I also just lost a craft due to some physics issues caused by IR, so I'm just going to close this PR.